profile/README.md

```diff
@@ -0,0 +1,5 @@
+## Profiling of Kubearmor Logs using karmor
+
+`karmor profile` shows a Terminal user interface table of three different Operations going on in Kubearmor: Process, File and Network. It maintains a counter of each operation that is happening within the cluster, along with other useful details. It directly fetches data from the `karmor log` API and displays all the needed information. The TUI includes simple navigation between operations and a user input based filter as well. 
```

Comment:
karmor profile which shows real-time terminal user interface table of three different operations going on in KubeArmor: Process, File and Network. It maintains a counter of each operation that is happening within the cluster, along with other useful details. It directly fetches data from the `karmor logs` API and displays all the required information. The TUI includes simple navigation between operations and a user input based filter as well.

fixed some letter case and some spellings